ui/components/multichain/account-details/account-details-section.tsx

```diff
@@ -36,12 +36,12 @@ export const AccountDetailsSection = ({
   const account = useSelector((state) =>
     getInternalAccountByAddress(state, address),
   );
-  const {
-    metadata: { keyring },
-  } = account;
+  const keyring = account?.metadata?.keyring;
```

Comment:
Is the optional chaining because of a lint error? the properties `metadata` and `keyring` should always be present in the `account` object